src/icons/icons/svgs/AppleIcon.tsx

```diff
@@ -3,8 +3,9 @@ import * as React from 'react';
 function SvgAppleIcon(props: React.SVGProps<SVGSVGElement>) {
   return (
     <svg
-      width={14}
+      width={16}
       height={16}
+      viewBox="0 0 14 16"
```

Comment:
this looks inconsistent with all the other usages, we should try to get from design a 16x16 svg

Reply:
`viewBox="0 0 14 16"` helps keep the icon centered. Here's an example of the PlanIcon which has the same dimensions. 

<img width="288" alt="Screen Shot 2020-12-14 at 9 24 25 AM" src="https://user-images.githubusercontent.com/13544620/102099931-79ce3500-3dee-11eb-8593-7d76c32042ae.png">

I think the `viewBox="0 0 16 16"` deviations--since they're pulled directly from Figma--are fine to keep unless QA in usage notes that the positioning looks weird.